Squircle/Squircle.swift

```diff
@@ -26,8 +26,24 @@ extension UIView {
     public func squircleWithBorder(width: CGFloat, color: UIColor) {
         self.layer.applySquircleWithBorder(width: width, color: color)
     }
+    
+    /**
+     Remove squircle corner radius and border.
+     */
+    public func removeSquircle() {
+        self.layer.removeSquircle()
+    }
+    
+    /**
+     Remove squircle border.
+     */
+    public func removeSquircleBorder() {
+        self.layer.removeSquircleBorder()
+    }
 }
 
+let squircleBorderName = "squircleBorder"
```

Comment:
Can this variable be internal? We don't want to expose it to the final user
```suggestion
internal let squircleBorderName = "squircleBorder"
```

Reply:
Makes perfect sense, but my test makes use of it. Use the string in the test instead?

Reply:
If the test file is importing target with `@testable import` it will have access to internal var and methods too